liquidity/liquidity.go

```diff
@@ -175,11 +348,240 @@ func (m *Manager) SuggestSwaps(ctx context.Context) (
 		suggestion := rule.suggestSwap(balance, outRestrictions)
 
 		// We can have nil suggestions in the case where no action is
-		// required, so only add non-nil suggestions.
-		if suggestion != nil {
-			suggestions = append(suggestions, suggestion)
+		// required, so we skip over them.
+		if suggestion == nil {
+			continue
+		}
+
+		// Get a quote for a swap of this amount.
+		// TODO(carla): add support for non-fast swaps.
+		quote, err := m.cfg.LoopOutQuote(
+			ctx, &loop.LoopOutQuoteRequest{
+				Amount:                  suggestion.Amount,
+				SweepConfTarget:         m.params.ConfTarget,
+				SwapPublicationDeadline: m.cfg.Clock.Now(),
+			},
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		// Our quote does not provide fee estimates for our off chain
+		// routing expenses, so we estimate them separately.
+		prePayFee, routeFee := m.estimateOffChain(
+			quote.PrepayAmount, suggestion.Amount,
+		)
+
+		log.Infof("quote for suggestion: %v, swap fee: %v, "+
```

Comment:
`Debugf`? Not sure

Reply:
Everything else needs debug, going to downgrade then make a note in the release notes recommended debug logging if you're using suggestions.